### PR TITLE
adding ensemble calibrate example

### PR DESCRIPTION
**Pull request by sabinala**
This PR adds an example of `ensemble_calibrate` to the interfaces notebook. This PR should be merged instead of #535.

### CODE REVIEW

File: docs/source/interfaces.ipynb
Comment (SamWitty) [inline docs/source/interfaces.ipynb:1]: Can you make sure the style of interfaces imports are consistent. This adds `from pyciemss.interfaces import ensemble_calibrate`, but the others call `pyciemss.ensemble_sample`, etc.

**Review by SamWitty — CHANGES_REQUESTED** [review]
Looks good except one minor comment.

File: docs/source/interfaces.ipynb
Comment (sabinala) [reply docs/source/interfaces.ipynb:1]: This is the error message I get when I don't include `from pyciemss.interfaces import ensemble_calibrate`, just `import pyciemss` and use `pyciemss.ensemble_calibrate`. I already tried `pip install -e .` and that didn't fix it...any advice?

```python
num_iterations = 10 if smoke_test else 100
model_paths = [model1, model2]
solution_mappings = [lambda x : x, lambda x : x]
data_mapping = {'case': 'infected', 'hosp': 'hospitalized'}
calibrated_results = pyciemss.ensemble_calibrate(model_paths, solution_mappings, dataset1, data_mapping=data_mapping, num_iterations=num_iterations)
parameter_estimates = calibrated_results["inferred_parameters"]
print(parameter_estimates())

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[5], line 5
      3 solution_mappings = [lambda x : x, lambda x : x]
      4 data_mapping = {'case': 'infected', 'hosp': 'hospitalized'}
----> 5 calibrated_results = pyciemss.ensemble_calibrate(model_paths, solution_mappings, dataset1, data_mapping=data_mapping, num_iterations=num_iterations)
      6 parameter_estimates = calibrated_results["inferred_parameters"]
      7 print(parameter_estimates())

AttributeError: module 'pyciemss' has no attribute 'ensemble_calibrate'
```

Comment (SamWitty) [reply docs/source/interfaces.ipynb:1]: @sabinala , try it again now that I've pushed in [c169bf4](https://github.com/ciemss/pyciemss/pull/537/commits/c169bf4bba07e1191b4cfe40c9ae573f74b0b947).

For future reference: https://docs.python.org/3/tutorial/modules.html

Comment (sabinala) [reply docs/source/interfaces.ipynb:1]: Thank you! Working now.

**Review by SamWitty — APPROVED** [review]

**Pull request merged**